### ISSUE TITLE
Expected SA1116 for single argument spanning lines

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1619]
**Source:**

```csharp
Func<Task<List<object>>, Task<string>> continuationFunction = task =>
    Task.Factory.StartNew(() =>
    {
        executed = true;
        return result;
    });
```

**Expected:**

I expected SA1116 to be reported for the `() =>` part of the argument to `StartNew`.

**Actual:**

SA1116 was not reported.

**StyleCop Classic behavior for this input:**

Unknown.

**Comment by vweijsters** [comment]
StyleCop classic does not report any diagnostic for this fragment

**Comment by Maxwe11** [comment]
It shouldn't be reported. Rule applicable only for methods with more than one parameter/argument.
